src/test/java/com/google/javascript/clutz/DeclarationGeneratorTests.java

```diff
@@ -130,7 +131,9 @@ static String getExternFileNameOrNull(String testFileName) {
             .flatMap(List::stream)
             .map(Path::toFile);
 
-    return Stream.concat(jsFiles, zipFiles).collect(Collectors.toList());
+    List<File> fileList = Stream.concat(jsFiles, zipFiles).collect(Collectors.toList());
+    Collections.sort(fileList);
+    return fileList;
```

Comment:
I don't get this part, ... ?

Reply:
This was just to make test execution reproducible and make it easier to alphabetically go through the list of failures.